### ISSUE TITLE
Add --silent option to not output file upon success

### ISSUE THREAD

**Issue by mosbth** [nodeca/js-yaml#174]
Consider to add `--silent, -s` option to `bin/js-yaml.js` for silent output upon success. It makes it easier to parse a lot of files, for example from a bash-script, and only focus on those giving errors.

Something like this in `bin/js-yaml.js` would fix it.

```
cli.addArgument(['-s', '--silent'], {
  help:   'Do not output file upon success',
  action: 'storeTrue'
});

if (!options.silent) {
  if (isYaml) {
    console.log(JSON.stringify(output, null, '  '));
  } else {
    console.log(yaml.dump(output));
  }    
}
```

**Comment by puzrin** [comment]
Why not use stdout redirect `> /dev/null` ?

**Comment by mosbth** [comment]
Yes, I can use stdout redirection. Thats fine if thats the preferred way for "silent output".

The reason was that I have a large script using several validator tools (for html, css, js, php, python, etc). All of them I could use in a silent manner, only focusing on the error produced. `bin/js-yaml.js` was the first validator I encountered doing output on success and I saw no way of removing the output. `--silent` would do the trick, but I can live with stdout redirection also.
